<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>S_UserManager</class>
 <widget class="QDialog" name="S_UserManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
#frame_3
{
		background-color: qlineargradient(spread:pad, x1:0.637, y1:0.279591, x2:0.489, y2:0.574, stop:0 rgba(255, 151, 0, 255), stop:0.75 rgba(255, 170, 0, 255));
}

QToolButton
{
 
    
    border-color: #1e1e1e;
    border-style: solid;
    border-radius: 6;
    padding: 3px;
    font-size: 12px;
    padding-left: 5px;
    padding-right: 5px;	
	
	
	font: 75 18pt &quot;Cambria Math&quot;;
	color: rgb(255, 255, 255);
}

QToolButton:hover
{
    border: 4px solid  qlineargradient(spread:reflect, x1:0.653, y1:0.699136, x2:0.437227, y2:0.057, stop:0.159091 rgba(59, 59, 59, 255), stop:0.846591 rgba(143, 143, 143, 255));
}

#Product{
	
	background-color: qlineargradient(spread:reflect, x1:0.653, y1:0.699136, x2:0.437227, y2:0.057, stop:0.159091 rgba(59, 59, 59, 255), stop:0.846591 rgba(143, 143, 143, 255));
}
#btnDelete{
	background-color: qlineargradient(spread:pad, x1:0.528, y1:0.749227, x2:0.795, y2:0.209818, stop:0.1875 rgba(218, 0, 0, 255), stop:0.818182 rgba(232, 0, 0, 255));

}
#btnNew{
background-color: qlineargradient(spread:pad, x1:0.637, y1:0.279591, x2:0.489, y2:0.574, stop:0 rgba(255, 151, 0, 255), stop:0.75 rgba(255, 170, 0, 255));
	
}
#btnMenu{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.152818, x2:1, y2:0.932, stop:0 rgba(85, 170, 0, 255), stop:0.573864 rgba(47, 95, 0, 255));
}
</string>
  </property>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>481</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame_3
{
		background-color: qlineargradient(spread:pad, x1:0.637, y1:0.279591, x2:0.489, y2:0.574, stop:0 rgba(255, 151, 0, 255), stop:0.75 rgba(255, 170, 0, 255));
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QToolButton" name="btnBack">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton
{
 
    
    border-color: #1e1e1e;
    border-style: solid;
    border-radius: 6;
    padding: 3px;
    font-size: 12px;
    padding-left: 5px;
    padding-right: 5px;	
	
	
	font: 75 18pt &quot;Cambria Math&quot;;
	color: rgb(255, 255, 255);
}
QToolButton:pressed{
	
	background-color: qlineargradient(spread:pad, x1:0.067, y1:0.153, x2:0.466, y2:0.567136, stop:0.1875 rgba(255, 170, 0, 255), stop:0.818182 rgba(245, 163, 0, 255));
}
QToolButton:hover{
	    border: 4px solid  qlineargradient(spread:pad, x1:0.255, y1:0.982955, x2:0.852182, y2:0.232, stop:0.1875 rgba(255, 170, 0, 0), stop:0.818182 rgba(255, 255, 255, 0))
}</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="images.qrc">
      <normaloff>:/images/images/Back_icon.png</normaloff>:/images/images/Back_icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>120</width>
      <height>120</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="ledSearch">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> QLineEdit {
    	border: 4px qlineargradient(spread:pad, x1:0.63, y1:0.829546, 			x2:0.852182, y2:0.232, stop:0 rgba(255, 170, 0, 0), stop:1 rgba(255, 220, 19, 0));
    	border-radius: 10px;
		
    	background:qlineargradient(spread:pad, x1:0.63, y1:0.829546, x2:0.852182, y2:0.232, stop:0 rgba(255, 170, 0, 255), stop:1 rgba(255, 220, 19, 255)) ;
		
	font: 20pt &quot;MS Shell Dlg 2&quot;;
    padding: 1px 2px 1px 40px;
	color: rgb(255, 255, 255);
    }
QLineEdit:pressed{
	padding-right:12px;
}
 </string>
    </property>
    <property name="placeholderText">
     <string>Search .....</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="cbbSearchType">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>0</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	/*text-align: center;*/
}
QComboBox QAbstractItemView
{
    border: 2px solid darkgray;
    selection-background-color: qlineargradient(spread:pad, x1:0.63, y1:0.829546, x2:0.852182, y2:0.232, stop:0 rgba(255, 170, 0, 255), stop:1 rgba(255, 220, 19, 255)) ;
text-align: center;
}
QComboBox::down-arrow
{
	
	
	image: url(:/images/images/down-arrow.png);
}
QComboBox:hover,QPushButton:hover {
     border: 2px solid qlineargradient(spread:pad, x1:0.63, y1:0.829546, x2:0.852182, y2:0.232, stop:0 rgba(255, 170, 0, 255), stop:1 rgba(255, 220, 19, 255)) ;
 }

    QComboBox {
    	border: 4px qlineargradient(spread:pad, x1:0.63, y1:0.829546, 			x2:0.852182, y2:0.232, stop:0 rgba(255, 170, 0, 0), stop:1 rgba(255, 220, 19, 0));
    	
		
    	background:qlineargradient(spread:pad, x1:0.63, y1:0.829546, x2:0.852182, y2:0.232, stop:0 rgba(255, 170, 0, 255), stop:1 rgba(255, 220, 19, 255)) ;
		
	height:40px;
	width:350px;
	font: 75 11pt &quot;MS Shell Dlg 2&quot;;
    padding: 1px 2px 1px 10px;
   	text-align: left;
    }
  QComboBox::drop-down
{
     subcontrol-origin: padding;
     subcontrol-position: top right;
     width: 15px;

     border-left-width: 0px;
     border-left-color: darkgray;
     border-left-style: solid; /* just a single line */
     border-top-right-radius: 3px; /* same radius as the QComboBox */
     border-bottom-right-radius: 3px;
 }
 
     
     
    </string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>280</y>
     <width>481</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cambria Math</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QToolButton" name="btnMenu">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="images.qrc">
      <normaloff>:/images/images/menu.png</normaloff>:/images/images/menu.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>200</width>
      <height>200</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="btnNew">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="images.qrc">
      <normaloff>:/images/images/AddNew.png</normaloff>:/images/images/AddNew.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>300</width>
      <height>300</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="btnDelete">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>0</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0.528, y1:0.749227, x2:0.795, y2:0.209818, stop:0.1875 rgba(218, 0, 0, 255), stop:0.818182 rgba(232, 0, 0, 255));</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="images.qrc">
      <normaloff>:/images/images/report.png</normaloff>:/images/images/report.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>200</width>
      <height>200</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QTableWidget" name="tblListUser">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>480</width>
     <height>240</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(0, 0, 255);</string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::EditKeyPressed</set>
   </property>
   <property name="tabKeyNavigation">
    <bool>true</bool>
   </property>
   <property name="showDropIndicator" stdset="0">
    <bool>true</bool>
   </property>
   <property name="dragDropOverwriteMode">
    <bool>false</bool>
   </property>
   <property name="showGrid">
    <bool>true</bool>
   </property>
   <property name="gridStyle">
    <enum>Qt::DashDotLine</enum>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>120</number>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>FisrtName</string>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>LastName</string>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Role</string>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Phone</string>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string/>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
